.github/workflows/build.yml

```diff
@@ -6,6 +6,7 @@ on:
       - opened
       - reopened
       - synchronize
+      - labeled
```

Comment:
Can this be limited to just the SafeToBuild label being added? There is a ton of labels we add and we don't want a new build every time we add labels.

Reply:
Maybe. I saw some examples of looking for specific labels, but I don't think it was up the type section for `pull_request_target`. Checking now to see if there exists a construction for that...